obs-browser-source.cpp

```diff
@@ -134,14 +134,21 @@ bool BrowserSource::CreateBrowser()
 		if (hwaccel) {
 			obs_enter_graphics();
 			tex_sharing_avail = gs_shared_texture_available();
+#if CHROME_VERSION_BUILD >= 4430 && defined(__APPLE__) && defined(__arm64__)
+			// CEF 4430 doesn't seem to work correctly when init'd with shared texture on arm64:
+			//  - we never see OnAcceleratedPaint called
+			//  - everything is black
+			// so we'll disable that for the time being
```

Comment:
There was a comment on CEF PR 285 recently about OnPaint was being called instead of OnAcceleratedPaint and they had to disable the sandbox to get this to work.
https://bitbucket.org/chromiumembedded/cef/pull-requests/285/reimplement-shared-texture-support-for-viz#comment-243274341

Reply:
Interesting, nice find, will take a look!  Not sure if you want potential M1 users to take a look at Apple silicon OBS Browser with/without this change.  Won't be able to get to it today.